index.js

```diff
@@ -161,4 +175,26 @@ function cartItem() {
   
   function calculate(totalPrice) {
     totalNumber.innerText = "£" + totalPrice.toFixed(2);
-  }
\ No newline at end of file
+  }
+  function filter (){
```

Comment:
What is this function does? can you explain briefly please? very interesting for me.